src/python/grpcio_tests/tests/tests.json

```diff
@@ -25,7 +25,8 @@
   "_implementations_test.CallCredentialsTest",
   "_implementations_test.ChannelCredentialsTest",
   "_insecure_interop_test.InsecureInteropTest",
-  "_invalid_metadata_test.InvalidMetadataTest"
+  "_invalid_metadata_test.InvalidMetadataTest",
```

Comment:
@nathanielmanistaatgoogle It's an artefact of the string missing a comma delimiter at the end of the line on the current v1.0.x head.